po/it_IT.po

```diff
@@ -16,188 +16,405 @@ msgstr ""
-msgstr "Joystick destro"
+#: inputremapper/gui/components/editor.py:65
+msgid "Record the input first"
+msgstr ""
+
+#: data/input-remapper.glade:1708
+msgid ""
+"Release all inputs which are part of the combination before the mapping is "
+"injected"
+msgstr ""
+
+#: data/input-remapper.glade:1711
+msgid "Release input"
+msgstr ""
+
+#: data/input-remapper.glade:1683
+msgid "Release timeout"
+msgstr ""
+
+#: inputremapper/gui/controller.py:208
+msgid "Remove the Analog Output Axis when specifying a macro or key output"
+msgstr ""
+
+#: inputremapper/gui/controller.py:199
+msgid ""
+"Remove the macro or key from the macro input field when specifying an analog "
+"output"
+msgstr ""
 
-#: data/input-remapper.glade:368
+#: data/input-remapper.glade:1776
+msgid "Remove this input"
+msgstr ""
+
+#: data/input-remapper.glade:394
+msgid "Rename"
+msgstr "Rinomina"
+
+#: data/input-remapper.glade:452
 msgid "Save the entered name"
 msgstr "Salva il nome inserito"
 
-#: data/input-remapper.glade:124
+#: data/input-remapper.glade:1449
 msgid ""
-"Shortcut: ctrl + del\n"
-"To give your keys back their original mapping."
+"See <a href=\"https://github.com/sezanzeb/input-remapper/blob/HEAD/readme/"
+"usage.md\">usage.md</a> online on github for comprehensive information.\n"
+"\n"
+"A \"key + key + ... + key\" syntax can be used to trigger key combinations. "
+"For example \"Control_L + a\".\n"
+"\n"
+"Writing \"disable\" as a mapping disables a key.\n"
+"\n"
+"Macros allow multiple characters to be written with a single key-press. "
+"Information about programming them is available online on github. See <a "
+"href=\"https://github.com/sezanzeb/input-remapper/blob/HEAD/readme/macros."
+"md\">macros.md</a> and <a href=\"https://github.com/sezanzeb/input-remapper/"
+"blob/HEAD/readme/examples.md\">examples.md</a>"
 msgstr ""
-"Scorciatoia: ctrl + del\n"
-"Per restituire alle tue chiavi la loro mappatura originale."
 
-#: data/input-remapper.glade:1556
+#: data/input-remapper.glade:1590
 msgid "Shortcuts"
 msgstr "Scorciatoie"
 
-#: data/input-remapper.glade:1458
+#: data/input-remapper.glade:1492
 msgid ""
 "Shortcuts only work while keys are not being recorded and the gui is in "
 "focus."
 msgstr ""
-"Le scorciatoie funzionano solo mentre i tasti non vengono registrati e la GUI è in "
-"messa a fuoco."
+"Le scorciatoie funzionano solo mentre i tasti non vengono registrati e la "
+"GUI è in messa a fuoco."
+
+#: data/input-remapper.glade:322
+#, fuzzy
+msgid "Start injecting. Don't hold down any keys while the injection starts"
+msgstr "Non tenere premuto alcun tasto durante l'avvio dell'iniezione."
+
+#: inputremapper/gui/controller.py:643
+#, fuzzy
+msgid "Starting injection..."
+msgstr "ferma l'iniezione"
+
+#: data/input-remapper.glade:201 data/input-remapper.glade:334
+msgid "Stop"
+msgstr "Ripristina impostazioni predefinite"
+
+#: inputremapper/gui/controller.py:710
+#, fuzzy
+msgid "Stopped the injection"
+msgstr "ferma l'iniezione"
```

Comment:
Thanks!

Do you know why `"Starting injection..."` and `"Stopped the injection"` have the same msgstr here?

Reply:
That's not harmful, so it is safe to leave as is.

When using `msgmerge` to update a PO file with an updated POT file, Gettext does a fuzzy match of the new strings against old strings in the PO file. When a fuzzy match is found, that new string receives the content of the previous plus a `#, fuzzy` tag.

That 'fuzzy' tag means the translation needs to be reviewed by a translator. Until then, that translation string will not show up in the translated UI.

For translators it is good because they might reuse it when updating the language translation. For the built software, no impact at all.